src/Models/SingleColumnModels/single_column_model.jl

```diff
@@ -51,45 +51,54 @@ function Models.make_ode_function(model::SingleColumnModel{FT}) where {FT}
         w = Ym.w
         ρθ = Ym.ρθ
 
+        # gather boundary conditions
+        bc_ρ = model.boundary_conditions.ρ
+        bc_uv = model.boundary_conditions.uv
+        bc_w = model.boundary_conditions.w
+        bc_ρθ = model.boundary_conditions.ρθ
+
         # density
+        flux_bottom = get_boundary_flux(model, bc_ρ.bottom, ρ, Ym, Ya)
+        flux_top = get_boundary_flux(model, bc_ρ.top, ρ, Ym, Ya)
         If = Operators.InterpolateC2F()
         ∂f = Operators.GradientC2F()
         ∂c = Operators.DivergenceF2C(
-            bottom = Operators.SetValue(Geometry.Cartesian3Vector(zero(FT))),
-            top = Operators.SetValue(Geometry.Cartesian3Vector(zero(FT))),
+            bottom = Operators.SetValue(flux_bottom),
+            top = Operators.SetValue(flux_top),
         )
         @. dρ = -∂c(w * If(ρ))
 
         # potential temperature
+        flux_bottom = get_boundary_flux(model, bc_ρθ.bottom, ρθ, Ym, Ya)
+        flux_top = get_boundary_flux(model, bc_ρθ.top, ρθ, Ym, Ya)
         If = Operators.InterpolateC2F()
         ∂f = Operators.GradientC2F()
         ∂c = Operators.DivergenceF2C(
-            bottom = Operators.SetValue(Geometry.Cartesian3Vector(zero(FT))),
-            top = Operators.SetValue(Geometry.Cartesian3Vector(zero(FT))),
+            bottom = Operators.SetValue(flux_bottom),
+            top = Operators.SetValue(flux_top),
         )
         # TODO!: Undesirable casting to vector required
         @. dρθ =
             -∂c(w * If(ρθ)) + ρ * ∂c(Geometry.CartesianVector(ν * ∂f(ρθ / ρ)))
 
-        uv_1 = Operators.getidx(uv, Operators.Interior(), 1)
-        u_wind = LinearAlgebra.norm(uv_1)
-
         A = Operators.AdvectionC2C(
             bottom = Operators.SetValue(Geometry.Cartesian12Vector(0.0, 0.0)),
             top = Operators.SetValue(Geometry.Cartesian12Vector(0.0, 0.0)),
         )
 
         # uv
-        bcs_bottom = Operators.SetValue(
-            Geometry.Cartesian3Vector(Cd * u_wind) ⊗ uv_1,
-        )
-        bcs_top = Operators.SetValue(uvg)
+        flux_bottom = get_boundary_flux(model, bc_uv.bottom, uv, Ym, Ya)
+
+        bcs_bottom = Operators.SetValue(flux_bottom)
+        bcs_top = Operators.SetValue(uvg) # this needs abstraction
```

Comment:
This abstraction is something we can address as we consider changes to the interface i.e. application of boundary conditions to faces (associated with the `Domain`, and tagged at the `Model` level) @glwagner